.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: SODA Multicloud CI Build
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
why to restrict on branches? why not on: [push, pull_request]..allow for all

Reply:
This is still WIP. restricted for now. Will make this change.

Reply:
ok